pandas/tests/reshape/merge/test_merge_anti.py

```diff
@@ -0,0 +1,407 @@
+import numpy as np
+import pytest
+
+import pandas as pd
+from pandas import DataFrame
+import pandas._testing as tm
```

Comment:
from pandas import testing as tm

Reply:
Actually pre-commit is not allowing that.
![Screenshot 2021-08-19 at 6 54 57 PM](https://user-images.githubusercontent.com/6472290/130076442-c04adb4f-7511-465f-af16-d7c5a51c36db.png)
It also looks like this in other files
https://github.com/pandas-dev/pandas/blob/3af1a4fa27a7d48d8e6aea66bf2da6fdf2e1550e/pandas/tests/reshape/merge/test_merge.py#L34